tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -1767,6 +1767,14 @@ TEST (nnstreamer_capi_singleshot, invoke_timeout)
     EXPECT_EQ (status, ML_ERROR_TIMED_OUT);
     EXPECT_TRUE (output == NULL);
 
+    /* sleep 2 seconds (enough time to arrive the buffer in appsink) */
+    g_usleep (2000 * 1000);
```

Comment:
How did you calculate the 2seconds? Is this heuristic value?

Reply:
right, img classification model (tf-lite) takes less than 300 ms in my local.

Reply:
You can remove this ```g_usleep``` by invoking with ```output2``` in the next invoke.

Reply:
you are right, I will remove this block in next week.